torch/csrc/README.md

```diff
@@ -68,6 +75,27 @@ exception which doesn't contain any info, instead it says, "An error
 occurred in the Python API; if you return to the interpreter, Python
 will raise that exception, nothing else needs to be done."
 
+* For code that you bind using pybind, `HANDLE_TH_ERRORS` and `END_HANDLE_TH_ERRORS_PYBIND`
+can be used. They will work jointly with pybind error handling to raise
+pytorch errors and warnings natively and let pybind handle other errors. It can be used as:
+
+```
+// Function given to the pybind binding
+at::Tensor foo(at::Tensor x) {
+  HANDLE_TH_ERRORS
+  ...
+  if (!x) throw python_error();
+  // pybind native error
+  if (!x) throw py::value_error();
+  // From c10/Exception.h
+  TORCH_CHECK(cond, "cond was false here");
+  TORCH_WARN("Warning message");
+  ...
+  END_HANDLE_TH_ERRORS_PYBIND
```

Comment:
Sigh, it's gonna be so hard to get people to actually use this macro XD

Reply:
We could provide a `get_safe_version` like the one used in `load_inline` in the whole code base and recommend all pybind bindings to use `get_safe_version(foo)` instead of `foo` in their `m.def` ?